platform-api/src/main/java/io/datacater/core/kubernetes/DataCaterK8sConfig.java

```diff
@@ -48,4 +51,42 @@ private DataCaterK8sConfig() {}
       ConfigProvider.getConfig()
           .getOptionalValue("datacater.pythonrunner.replicas", Integer.class)
           .orElse(1);
+
+  private static final String CPU = "cpu";
+  private static final String MEMORY = "memory";
+  static final String PYTHONRUNNER_RESOURCES_REQUESTS_CPU =
+      ConfigProvider.getConfig()
+          .getOptionalValue("datacater.pythonrunner.resources.requests.cpu", String.class)
+          .orElse("0.1");
+  static final String PYTHONRUNNER_RESOURCES_REQUESTS_MEMORY =
+      ConfigProvider.getConfig()
+          .getOptionalValue("datacater.pythonrunner.resources.requests.memory", String.class)
+          .orElse("200Mi");
+  static final Optional<String> PYTHONRUNNER_RESOURCES_LIMITS_CPU =
+      ConfigProvider.getConfig()
+          .getOptionalValue("datacater.pythonrunner.resources.limits.cpu", String.class);
+  static final String PYTHONRUNNER_RESOURCES_LIMITS_MEMORY =
+      ConfigProvider.getConfig()
+          .getOptionalValue("datacater.pythonrunner.resources.requests.memory", String.class)
+          .orElse("0.5");
```

Comment:
`0.5` does not make sense as a default value for the memory limit. This sounds more like it was meant for CPU.

Reply:
Might just use the same default as for the request to follow the best practice of setting memory request=memory limit?